lms/models/region.py

```diff
@@ -18,25 +18,21 @@ class Region:
 class Regions:
     """A collection of all the regions as an enum like object."""
 
-    # As region objects are frozen, these can be directly compared like enums
-    US = Region(code="us", name="Worldwide (U.S.)", authority="lms.hypothes.is")
-    CA = Region(code="ca", name="Canada", authority="lms.ca.hypothes.is")
-    ALL = US, CA
-
-    _AUTHORITY_MAP = {region.authority: region for region in ALL}
-    _CODE_MAP = {region.code: region for region in ALL}
```

Comment:
Removing the hard-coded list of allowable authorities and the implicit assumption of a one:one mapping from region codes to authorities.